.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
```

Comment:
Seems that my suggestions are having backticks removed due to the suggestion using backticks:
```suggestion
**Describe the solution you'd like**
A clear and concise description of what you want to happen.

**Please include an example where applicable:**
``` text
original text
```

``` text
expected outcome
```
```

Reply:
The suggestion looks like so:

**Describe the solution you'd like**
A clear and concise description of what you want to happen.

**Please include an example where applicable:**
``` text
original text
```

``` text
expected outcome
```